subprojects/core/src/main/java/org/gradle/api/internal/tasks/options/AbstractOptionElement.java

```diff
@@ -32,7 +33,7 @@
     private final Class<?> optionType;
     private final NotationParser<CharSequence, ?> notationParser;
 
-    public AbstractOptionElement(String optionName, Option option, Class<?> optionType, Class<?> declaringClass, NotationParser<CharSequence, ?> notationParser) {
+    AbstractOptionElement(String optionName, Option option, Class<?> optionType, Class<?> declaringClass, NotationParser<CharSequence, ?> notationParser) {
```

Comment:
Force of habit really. I always try to minimise visibility and mark anything with only private constructors as final as it can't be sub-classes anyway. I can revert those changes if it doesn't fit with Gradle's code style